src/main/java/budgetbuddy/storage/StorageManager.java

```diff
@@ -24,15 +26,17 @@
 
     private static final Logger logger = LogsCenter.getLogger(StorageManager.class);
     private LoansStorage loansStorage;
+    private AccountsStorage accountsStorage;
     private RuleStorage ruleStorage;
     private ScriptsStorage scriptsStorage;
     private UserPrefsStorage userPrefsStorage;
 
 
-    public StorageManager(LoansStorage loansStorage, RuleStorage ruleStorage,
+    public StorageManager(LoansStorage loansStorage, RuleStorage ruleStorage, AccountsStorage accountsStorage,
```

Comment:
For standardisation purposes I think the order of the parameters matter. In the previous files the order has been loans-accounts-rules-scripts. So perhaps you might want to put accountsstorage in front of rulestorage here as well